build.gradle

```diff
@@ -103,6 +103,9 @@ rat {
             'node_modules/**',
             '.idea/**',
             'README.md',
+            'docker/Dockerfile',
+            'docker/docker-compose.yml',
+            'docker/assets/',
```

Comment:
He originally had that, but I have him grabbing the latest release instead.  We could have two versions of this...